### ISSUE TITLE
check Windows build and re-enable Windows tests

### ISSUE THREAD

**Issue by KrisThielemans** [conda-forge/stir-feedstock#42]
#39 disabled windows tests due to failures, probably related to `STIR_CONFIG_DIR`. It is likely that the `activate` scripts don't work. Testing currently has to be done manually therefore.

**Comment by KrisThielemans** [comment]
Possibly the following might work on Windows
```
conda env config vars set STIR_CONFIG_DIR=%CONDA_PREFIX%\Library\share\stir\config
```
until we fix this.

See also #44

**Comment by KrisThielemans** [comment]
fixed with #46
